docs/koji.md

```diff
@@ -61,6 +62,8 @@ Example type-specific content generator metadata in context:
     "release": "1",
     "extra": {
       "image": {
+        "autorebuild": "False",
+        "help": None,
```

Comment:
Ah, good catch! Yes, we need to update this. Probably should read like this though:
```
{
  "autorebuild": false,
  "help": null
},
```
(no comma after the final entry)